src/Shared/Recruit.Vacancies.Client/Application/CommandHandlers/LiveVacancyCommandHandler.cs

```diff
@@ -7,28 +7,38 @@
 using System.Threading.Tasks;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Esfa.Recruit.Vacancies.Client.Domain.Services;
+using Microsoft.Extensions.Logging;
 
 namespace Esfa.Recruit.Vacancies.Client.Application.CommandHandlers
 {
     public class LiveVacancyCommandHandler : IRequestHandler<LiveVacancyCommand>
     {
+        private readonly ILogger<LiveVacancyCommandHandler> _logger;
         private readonly IVacancyRepository _repository;
         private readonly IMessaging _messaging;
         private readonly ITimeProvider _timeProvider;
 
-        public LiveVacancyCommandHandler(IVacancyRepository repository, IMessaging messaging, ITimeProvider timeProvider)
+        public LiveVacancyCommandHandler(
+            ILogger<LiveVacancyCommandHandler> logger,
+            IVacancyRepository repository, 
+            IMessaging messaging, 
+            ITimeProvider timeProvider)
         {
+            _logger = logger;
             _repository = repository;
             _messaging = messaging;
             _timeProvider = timeProvider;
         }
 
         public async Task Handle(LiveVacancyCommand message, CancellationToken cancellationToken)
         {
+            _logger.LogInformation("Setting vacancy Live for vacancy {vacancyReference}.", message.VacancyId);
```

Comment:
`vacancyId` :( I know what you mean, it is confusing using Id's and referencenumbers in different places. Should we just stick to using VacancyId, because at least that is always there. It mean we will have ti do a lookup when we are investigating/stepping through. I know this is to do with the QA process where we surface the reference number (not the id)

Reply:
👍